users/getAllUsers.js

```diff
@@ -1,7 +1,7 @@
 const UsersDb = require('./users-model');
 
 function getAllUsers(req, res) {
-  UsersDb.findAllUsers()
+  UsersDb.find()
```

Comment:
Your controller is missing a catch block in case getting all users don't work.

Reply:
Done in last commit